src/python/pants/engine/target.py

```diff
@@ -119,6 +131,13 @@ def compute_value(cls, raw_value: Optional[int], address: Address) -> Optional[i
                 return value_or_default
     """
 
+    # Opt-in per field class to use a "missing value" marker for the `raw_value` in
+    # `compute_value()` in case the field was not represented in the BUILD file.
+    #
+    # This will allow users to provide `None` as the field value (when applicable) without getting
+    # the fields default value.
+    use_no_value: ClassVar[bool] = False
```

Comment:
Should this be named something like `none_is_valid_value`? I think that expresses what it's for? I.e., that a user-provided `None` means something other than "use the default"?

Reply:
I'm fine with `none_is_valid_value`. :)